examples/ios_app/test/fixtures/bwb.xcodeproj/project.pbxproj

```diff
@@ -1719,6 +1719,7 @@
 				ENABLE_TESTABILITY = YES;
 				ENABLE_TESTING_SEARCH_PATHS = YES;
 				GCC_OPTIMIZATION_LEVEL = 0;
+				GCC_PREPROCESSOR_DEFINITIONS = AWESOME;
```

Comment:
It makes sense that these are back, and it was actually a bug that they were missing before.